deployment/terraform/modules/k8s-addons/traefik/templates/traefik-yaml.tpl

```diff
@@ -163,21 +163,4 @@ spec:
          backend:
            serviceName: http-span-collector
            servicePort: 80
----
-apiVersion: extensions/v1beta1
-kind: Ingress
-metadata:
-  name: traefik-pitchfork
```

Comment:
why did we remove from here?

Reply:
to create that ingress rule only if the pitchfork is enabled